src/io/flutter/inspector/InspectorService.java

```diff
@@ -352,19 +355,48 @@ private void onVmServiceReceived(String streamId, Event event) {
     }
   }
 
-  CompletableFuture<InstanceRef> invokeServiceMethodObservatoryNoGroup(String methodName) {
+  CompletableFuture<InstanceRef> invokeEvalNoGroup(String methodName) {
     return getInspectorLibrary().eval("WidgetInspectorService.instance." + methodName + "()", null, null);
   }
 
-  CompletableFuture<JsonElement> invokeServiceMethodDaemonNoGroup(String methodName, Map<String, Object> params) {
-    return getApp().callServiceExtension(ServiceExtensions.inspectorPrefix + methodName, params)
-      .thenApply((JsonObject json) -> {
-        if (json.has("errorMessage")) {
-          String message = json.get("errorMessage").getAsString();
-          throw new RuntimeException(methodName + " -- " + message);
+  CompletableFuture<JsonElement> invokeServiceExtensionNoGroup(String methodName, JsonObject params) {
+    return invokeServiceExtensionHelper(methodName, params);
+  }
+
+  private CompletableFuture<JsonElement> invokeServiceExtensionHelper(String methodName, JsonObject params) {
```

Comment:
this is the one real method added. The rest is just changing Map<String, Object> to JsonObject and renaming methods.